bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/EspressoHouse.java

```diff
@@ -0,0 +1,129 @@
+        postData.add(new BasicNameValuePair("ctl00$ctl00$ctl00$ContentPlaceHolderDefault$ContentPlaceHolderDefault$LoginView1$LoginUser$UserName",
+                username));
+        postData.add(new BasicNameValuePair("ctl00$ctl00$ctl00$ContentPlaceHolderDefault$ContentPlaceHolderDefault$LoginView1$LoginUser$Password",
+                password));
+        HttpResponse httpResponse = urlopen.openAsHttpResponse("http://www.espressohouse.com/coffee-card/logga-inladda/", postData, true);
+        LoginPackage lp = new LoginPackage(urlopen, postData, null, API_URL);
+        if (httpResponse.getStatusLine().getStatusCode() == 302) {
+            lp.setIsLoggedIn(true);
+        }
+        return lp;
+    }
+
+    @Override
+    public Urllib login() throws LoginException, BankException {
+        try {
+            LoginPackage lp = preLogin();
+            if (!lp.isLoggedIn()) {
+                throw new LoginException(res.getText(R.string.invalid_username_password).toString());
+            }
+        } catch (IOException e) {
+            throw new BankException(e.getMessage());
+        }
+        return urlopen;
+    }
+
+    @Override
+    public void update() throws BankException, LoginException, BankChoiceException {
+        super.update();
+        if (username == null || password == null || username.length() == 0 || password.length() == 0) {
+            throw new LoginException(res.getText(R.string.invalid_username_password).toString());
+        }
+        urlopen = login();
+        try {
+            String response = urlopen.open(API_URL);
+            Document dResponse = Jsoup.parse(response);
+            String card = dResponse.select(".transactionCardNumber").first().text().trim();
+            String cardNo = card.split(":")[1].trim();
+            String balance = dResponse.select(".balanceAmount").first().text();
+            balance = balance.substring(0, balance.length() - 2);
+            Account a = new Account(card,
+                    Helpers.parseBalance(balance),
+                    cardNo, Account.REGULAR, "SEK");
+
+            List<Element> eBalance = dResponse.select(".lineTotalAmount");
+            List<Element> eTransaction = dResponse.select(".lineTime");
+            List<Transaction> transactions = new ArrayList<>();
+            for (int i = 0; i < eTransaction.size(); i++) {
+                String s = eTransaction.get(i).text();
+                balance = eBalance.get(i).text();
+                balance = balance.substring(2, balance.length() - 2);
+                transactions.add(new Transaction(s.substring(s.length() - 16, s.length() - 6),
+                        s.substring(0, s.length() - 16), Helpers.parseBalance(balance)));
+            }
+            a.setTransactions(transactions);
+            accounts.add(a);
+        } catch (Exception e) {
+            throw new BankException(e.getMessage());
+        }
+        super.updateComplete();
+    }
+}
```

Comment:
Not to be picky, but it would be great if you could add a new line at the end of the file.